.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEATURE REQUEST]"
+labels: feature
+assignees: ''
+
+---
+
```

Comment:
We should ask them something about their background, as it will help us understand their request

Reply:
I've added this in the format of "What is the use case for this feature?", with some help text pointing towards their background.